src/package/types.ts

```
@@ -0,0 +1,53 @@
+export type ConnectionConfig = {
+  provider: Provider;
+  sessionConfig: SessionConfig;
+};
+
+export interface OaiEvent {
+  type: string;
+  server_sent?: boolean;
+  timestamp?: string;
+  id?: string;
+  event_id?: string;
+  [key: string]: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)